llvm/lib/MC/MCStreamer.cpp

```diff
@@ -267,6 +267,25 @@ void MCStreamer::emitDwarfLocDirective(unsigned FileNo, unsigned Line,
                                   Discriminator);
 }
 
+void MCStreamer::emitDwarfLocLabelDirective(SMLoc Loc, StringRef Name) {
+  auto &ctx = getContext();
+
```

Comment:
unneeded blank lines in these variable declarations.

Currently, `MCStreamer.cpp` doesn't reference MCDwarfLoc or MCDwarfLineEntry.  Perhaps the main work should be done in MCDwarf.cpp

Reply:
Good point - Moved those to `MCDwarf.cpp`